drivers/backup/auth.go

```diff
@@ -213,12 +228,28 @@ func getKeycloakEndPoint(admin bool) (string, error) {
 }
 
 // GetPxBackupNamespace returns namespace of px-backup deployment.
-func GetPxBackupNamespace() string {
+func GetPxBackupNamespace() (string, error) {
 	ns, present := os.LookupEnv(pxBackupNamespace)
-	if !(present) {
-		return AdminTokenSecretNamespace
+	var allServices *corev1.ServiceList
+	var err error
+	if allServices, err = k8sCore.ListServices("", metav1.ListOptions{}); err != nil {
+		if present {
```

Comment:
Why are you returning error if the env variable is set?

Reply:
Rephrased the error message to make more sense. Anyways, I have discarded this logic as we do not expect the env variable to be set.